components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "wso2_apim_publisher",
+  "version": "1.0.0",
+  "description": "WSO2 API Manager Web Apps Publisher",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config webpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wso2/carbon-apimgt"
+  },
+  "author": "WSO2 Org",
+  "license": "Apache License 2.0",
+  "dependencies": {
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "caniuse-api": "^1.6.1",
+    "material-ui": "^0.18.0",
+    "prop-types": "^15.5.10",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "react-router": "^4.1.1",
+    "react-router-dom": "^4.1.1",
+    "react-tap-event-plugin": "^2.0.1",
+    "swagger-client": "^3.0.10",
+    "webpack": "^2.5.0"
```

Comment:
Thanks @anuruddhal for the suggestion , As I know [create-react-app](https://github.com/facebookincubator/create-react-app) tool is a bootstrapping tool for creating create apps with zero-config way.It wraps all the commonly used webpack.configs within `node_modules/react-scripts/` module and if we need to do a modification to those, we would need to [eject](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-eject) them out to their default locations.